src/time.cr

```diff
@@ -254,27 +263,35 @@ struct Time
   # :nodoc:
   MAX_SECONDS = 315537897599_i64
 
-  # `DayOfWeek` represents a day-of-week in the Gregorian calendar.
+  # `DayOfWeek` represents a day of the week in the Gregorian calendar.
   #
   # ```
   # time = Time.new(2016, 2, 15)
   # time.day_of_week # => Time::DayOfWeek::Monday
   # ```
   #
-  # Alternatively, you can use question methods:
+  # Each member is identified by its ordinal number starting from `Monday = 1`
+  # according to [ISO 8601](http://xml.coverpages.org/ISO-FDIS-8601.pdf).
   #
-  # ```
-  # time.friday? # => false
-  # time.monday? # => true
-  # ```
+  # `#value` returns this ordinal number. It can easily be converted to the also
+  # common numbering based on `Sunday = 0` using `value % 7`.
   enum DayOfWeek
-    Sunday
-    Monday
-    Tuesday
-    Wednesday
-    Thursday
-    Friday
-    Saturday
+    Monday    = 1
+    Tuesday   = 2
+    Wednesday = 3
+    Thursday  = 4
+    Friday    = 5
+    Saturday  = 6
+    Sunday    = 7
+
+    # Returns the day of week that has the given value, or raises if no such member exists.
+    #
+    # This method also accepts `0` to identify `Sunday` in order to be compliant
+    # with the `Sunday = 0` numbering. All other days are equal in both formats.
+    def self.from_value(value : Int32)
+      value = (value + 6) % 7 + 1
```

Comment:
surely `return Sunday if value == 0` would be better here?

Reply:
What should happen when the value is greater than `7`? 

The current implementation pass for the test below, is it the expected behavior?
```crystal
Time::DayOfWeek.from_value(8).should eq Time::DayOfWeek::Monday
Time::DayOfWeek.from_value(9).should eq Time::DayOfWeek::Tuesday
# and goes on...
```

Reply:
I think either makes some kind of sense. But we can make it more strict, no problem.